### PR TITLE
Warn against the "anonymous function in pipeline" trick

### PR DESCRIPTION
**Pull request by michalmuskala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -545,6 +545,31 @@
   # Good
   result = String.strip(input)
   ```
+  
+* <a name="anonymous-pipeline"></a>
+  Every time you use the "anonymous function in the pipeline" trick a puppy dies!
```
Comment (whatyouhide) [inline README.md]: I am personally a bit against the funny bit here and I'd just go with "Don't use anonymous bla bla bla". Also many users won't know what the *"anonymous function in the pipeline" trick* is, so I would rather say again just "Don't use anonymous functions in pipelines." or something similar.

Comment (whatyouhide) [reply README.md]: @michalmuskala ping.

**Comment by michalmuskala** [comment]
Updated

**Comment by whatyouhide** [comment]
Thanks @michalmuskala! 💟

**Pull request merged**